### ISSUE TITLE
chromebook_kukui: misc: External headphones are working on kappa.

### ISSUE THREAD

**Issue by aneeshlingala** [hexdump0815/imagebuilder#161]
I tried USB headphones and manually changed the output through pavucontrol. The sound output and headphone mic work perfectly.

**Comment by hexdump0815** [comment]
yes usb audio is working well, the internal mic is a bit more problematic and is only somewhat useable with some extra effort
